doc_admin/AdminManual.adoc

```diff
@@ -423,80 +431,36 @@ correctly perform any future Update process (see <<update-open-hospital,Update O
-** *rsc*
-** *sql*
-** CHANGELOG - Last version brief changelog
-** LICENSE - GNU GENERAL PUBLIC LICENSE V3
-** oh.ico - icon image for launcher
-** README.md
-** SetupGSM.cmd *Utility for Windows*
-** SetupGSM.sh *Utility for Linux*
-** startup.cmd *Launcher for Windows*
-** startup.sh *Launcher for Linux*
-
-=== 4.2 Folder tree in Open Hospital portable for Linux (32 and 64bit)
-
-In Open Hospital portable for Linux the following folder tree structure is found in the downloaded archive, where *_<version>_* is replaced with the current Open Hospital version and *_<arch>_* with the OS architecture (x32 or x64):
-
-* *OpenHospital-_<version>_-linux_i686-portable*/ or *OpenHospital-_<version>_-linux_x86_64-portable*/
-** data/ - folder for DB, DICOM files and logs
-** etc/ - working folder
-** _jdk-11.0.9+11-jre/_ (64bit) or _zulu11.43.21-ca-jre11.0.9-linux_i686/_ (32bit) - Java Virtual Machine
-** _mysql-<version>-<arch>-<build>/_ or _mariadb-<version>-<arch>/_ - MySQL Server or MariaDB Server
-** *oh/ the OpenHospital subfolder structure*
-** sql/ - working folder
-** CHANGELOG - Last version brief changelog
-** LICENSE - GNU GENERAL PUBLIC LICENSE V3
-** oh.ico - icon image for launcher
-** *oh.sh Launcher for OH-Linux*
-** POH-linux-changelog.md (changelog portable-linux)
-** POH-README.md (README portable-linux)
-
-=== 4.3 Folder tree in Open Hospital portable for Windows (32bit)
-
-In Open Hospital portable for Windows the following folder tree structure is found in the downloaded archive, where *_<version>_* is replaced with the current Open Hospital version:
-
-* *OpenHospital-_<version>_-win32-portable*/
-** data/ - folder for DB, DICOM files and logs
-** _jdk-11.0.9+11-jre/_ (64bit) or _zulu11.43.21-ca-jre11.0.9-linux_i686/_ (32bit) - Java Virtual Machine
+* *oh* – contains the Open Hospital application
+* *oh/bin* – contains the Open Hospital *binaries* (compiled software)
+* *oh/rpt* – contains the JasperReports® *reports* used in Open Hospital
+* *oh/rpt/PDF* – contains the reports generated by Open Hospital in PDF format
+* *oh/rsc* – contains the *resources* of Open Hospital, like configuration files and other related files
+* *oh/rsc/icons* – contains the *icons* used in Open Hospital for windows and buttons
+* *oh/rsc/images* – contains the *images* used in Open Hospital
+* *data* – contains all the data (database, logs) - created after the first startup
+* *data/db* – contains the database files
+* *data/logs* – contains all the logs
+* *data/dicom* – contains all the logs
+* *data/dump* – contains the backups
+* *sql* – contains the scripts and configurations for MySQL database
+* *tmp* – temporary folder
+
+These folders may be organized differently depending on the version of the software or the architecture of the operating system.
+
+Depending on the choosen operating system and architecture, you will also find the directories containing the needed external applications and libraries, like MySQL/MariaDB and Java JRE:
```

Comment:
Instead of `you will also find the directories containing the needed external applications and libraries, like MySQL/MariaDB and Java JRE:`  how about `directories containing external applications and libraries like MySQL/MariaDB and Java JRE that work with the chosen environment will be found:`    (or some similar wording)

Reply:
yes, absolutely